components/viewer/ViewProjectObj.vue

```diff
@@ -135,7 +135,7 @@ const isEnabled = computed<boolean>(() => {
   // Whether the object is always enabled or disabled based on the viewObject
   // Otherwise check the object conditions
   switch ($props.viewObject) {
-    case ViewContext.BackpackEnabled:
+    case ViewContext.BackpackEnabled || ViewContext.BackpackDisabled:
```

Comment:
That's not the right syntax, you need to use fall-through.

```suggestion
    case ViewContext.BackpackEnabled:
    case ViewContext.BackpackDisabled:
```